ios/FluentUI/Button/Button.swift

```diff
@@ -348,49 +268,17 @@ open class Button: UIButton {
         updateBackgroundColor()
         updateBorderColor()
 
-        layer.borderWidth = style.hasBorders ? Constants.borderWidth : 0
+        layer.borderWidth = tokenSet[.borderWidth].float
 
         if !isUsingCustomContentEdgeInsets {
-            edgeInsets = style.contentEdgeInsets
-        }
-
-        updateProposedTitleLabelWidth()
-    }
-
-    private func normalTitleAndImageColor(for window: UIWindow) -> UIColor {
-        if style.isFilledStyle {
-            return Colors.Button.titleWithFilledBackground
+            edgeInsets = defaultEdgeInsets()
         }
 
-        return style.isDangerStyle ? Colors.Palette.dangerPrimary.color : Colors.primary(for: window)
-    }
-
-    private func highlightedTitleAndImageColor(for window: UIWindow) -> UIColor {
-        if style.isFilledStyle {
-            return Colors.Button.titleWithFilledBackground
+        if #unavailable(iOS 15.0) {
+            titleLabel?.font = UIFont.fluent(tokenSet[.titleFont].fontInfo)
```

Comment:
should this be just part of updateTitleColors()? or updateTitleColors() function name needs to change to handle all types of label appearance update?